Tasks/AzureCLI/azureclitask.ts

```diff
@@ -30,8 +35,13 @@ export class azureclitask {
                 }
             }
             else {
-                var script:string = tl.getInput("inlineScript", true);
-                scriptPath = path.join(os.tmpdir() ,"azureclitaskscript.sh");
+                var script: string = tl.getInput("inlineScript", true);
+                if (scriptType === "bash") {
```

Comment:
You don't need this input.  Lib has os type.  On linux you should just create sh and on windows create .cmd